# https://github.com/strongswan/strongswan/commit/98b133c54c5e3f66f46a5bb11c9b09d06fdc8469

wolfssl_kem.c:28:10: fatal error: wolfssl/wolfcrypt/mlkem.h: No such file or directory
   28 | #include <wolfssl/wolfcrypt/mlkem.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

---

--- a/src/libstrongswan/plugins/wolfssl/wolfssl_kem.c
+++ b/src/libstrongswan/plugins/wolfssl/wolfssl_kem.c
@@ -25,10 +25,7 @@
 
 #ifdef WOLFSSL_HAVE_MLKEM
 
-#include <wolfssl/wolfcrypt/mlkem.h>
-#ifdef WOLFSSL_WC_MLKEM
 #include <wolfssl/wolfcrypt/wc_mlkem.h>
-#endif
 
 typedef struct private_key_exchange_t private_key_exchange_t;
 
